src/tiled/mainwindow.cpp

```diff
@@ -508,6 +508,10 @@ MainWindow::MainWindow(QWidget *parent, Qt::WindowFlags flags)
         new QShortcut(key, this, SLOT(delete_()));
 #endif
 
+    Preferences *prefs = Preferences::instance();
```

Comment:
There is already a pointer to the preferences in this scope, called `preferences`.